public/dataset/index.htm

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
```

Comment:
All the quotation marks in this are coming in malformed for me. They look like they're special characters. 

While it doesn't hurt having this page here, looking it over, I think it would also be really useful to have it as the readme to the https://github.com/townhallproject/mutual-aid-hub-server, and we could include the csv over there too.

Reply:
@meganrm Okay, funky quotes fixed.  I thought about switching the HTML to indicate UTF-8 charset or similar, but opted to just replace the special character quotes with good ol' "s.  I also removed the section of the license which doesn't belong (same thing I described in the other comment above).

If you'd like this content to live somewhere else, are you planning to move it around yourself, or can you tell me where it should go?

Finally, since it's been a week or so, I re-ran the script to generate a new version of the .csv dataset.